app/main.py

```diff
@@ -2,0 +1,28 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
+    mv = command[0]
+    file_to_move = command[1]
+    destination = command[2].split("/")
+    if mv != "mv":
+        raise SyntaxError(f"Command {mv} not found")
+    if file_to_move != "".join(destination):
+        try:
+            with open(file_to_move, "r") as source_file:
+                data = source_file.read()
+            os.remove(file_to_move)
+        except FileNotFoundError:
+            raise FileNotFoundError(f"File {file_to_move} not found")
```

Comment:
It's a good practice to handle exceptions. However, the FileNotFoundError is already clear enough. So, this try-except block can be skipped.